nft-rfc-006.md

```diff
@@ -0,0 +1,860 @@
+its use.
+
+Requirement 8 -- Use with self-sovereign identity
+-------------------------------------------------
+
+IIDs must be able to work with emerging self-sovereign identity
+approaches, which allows individuals to manage identifiers and
+associated credentials without reliance on trusted third parties.
+
+Requirement 9 -- Use with confidential storage
+----------------------------------------------
+
+IIDs must be able to work with emerging approaches to confidential
+storage, aka Secure Data Storage or Encrypted Data Vaults. These
+services need cryptographically enabled capabilities for authorization
+and robust mechanisms for encryption.
+
+Requirements 10 -- Recognizability
+----------------------------------
+
+IIDs must be recognizable as such. These identifiers may have unique
+properties compared to other identification approaches. When used in
+various contexts, it should be readily discernible that the identifier
+is, in fact, an IID.
+
+Requirement 11 -- Multiple metadata representations 
+----------------------------------------------------
+
+IIDs are expected to be used in a wide variety of contexts, with
+commensurate variety in the appropriateness of different serializations.
+IIDs must be able to represent associated metadata in a variety of
+formats without losing rigor. This includes representations of
+associated rights and attributes.
+
+Requirement 12 -- Leverage existing tooling and infrastructure
+--------------------------------------------------------------
+
+To the extent possible, IIDs should enable the use of widespread and
+mature tools rather than requiring bespoke or relatively untested
+innovative approaches.
+
+Anti-Requirement 1 -- Human readability
+---------------------------------------
+
+These identifiers are designed to be cryptographically secured and
+independent of trusted third parties; IIDs are secure and decentralized.
+However, they are not designed to facilitate human readability or use.
+User-friendly naming schemes, if any, must operate at another level.
+
+Proposal in brief
+=================
+
+Define IIDs as a profile of Decentralized Identifiers (DIDs) and support
+the use of Verifiable Credentials (VCs) for assertions. DIDs and VCs are
+specifications managed by the World Wide Web Consortium. Further, define
+all IIDs to dereference to a representation of the token itself while
+enabling IID References and IID Resources as identifiers within the
+assets's namespace for rigorous assertions and retrieval.
+
+IIDs define a family of DID Methods, allowing different chains and
+approaches to realize both IID and and DID functionality. IID Methods
+specify how IID (and DID) functions are achieved for a given chain,
+allowing for different parties to define Methods for the same chain
+should alternative approaches make sense---there need not be a
+definitive IID Method for a given chain.
+
+IIDs resolve to IID Documents, interoperably based on DID Documents, for
```

Comment:
We use IID Documents here without specifying the structure, it may become less manageable than using DID documents. This really needs to be justified, e.g what are the differences between IID documents and DID documents, and what is the cost vs benefit associated with this new definition.

Reply:
Agreed. IID Documents *are* conformant DID Documents. I'll update the language to clarify.